src/hotspot/share/classfile/systemDictionaryShared.hpp

```diff
@@ -247,12 +247,8 @@ class SystemDictionaryShared: public SystemDictionary {
   static bool is_sharing_possible(ClassLoaderData* loader_data);
 
   static bool add_unregistered_class(InstanceKlass* k, TRAPS);
-  static InstanceKlass* dump_time_resolve_super_or_fail(Symbol* class_name,
-                                                Symbol* super_name,
-                                                Handle class_loader,
-                                                Handle protection_domain,
-                                                bool is_superclass,
-                                                TRAPS);
+  static InstanceKlass* lookup_super_for_unregistered_class(Symbol* class_name,
+                            Symbol* super_name,  bool is_superclass);
```

Comment:
Indentation is ad-hoc here, please align Symbol

Reply:
Fixed.